RISE-V2G-EVCC/src/main/java/com/v2gclarity/risev2g/evcc/transportLayer/TCPClient.java

```diff
@@ -148,6 +149,10 @@ public void stop() {
 		}
 	}
 	
+	private static void releaseInstance() {
+		uniqueTCPClientInstance = null;
+	}
+	
```

Comment:
We could also get rid of this method and just put this line of code into line 142 instead. I prefer less and cleaner code.

Reply:
no problem, reorganized this.